hyperdash/sdk_message.py

```diff
@@ -7,7 +7,17 @@
 TYPE_STARTED = 'run_started'
 TYPE_ENDED = 'run_ended'
 TYPE_HEARTBEAT = 'heartbeat'
+TYPE_METRIC = 'metric'
 
+def create_metric_message(sdk_run_uuid, value, name):
```

Comment:
super nit but you should swap the order of the value and name arguments to be consistent with the API exposed to the users as well